alvr/server/cpp/alvr_server/FakeViveTracker.cpp

```diff
@@ -0,0 +1,107 @@
+#include "FakeViveTracker.h"
+#include "Settings.h"
+
+#include <cassert>
+#include <format>
```

Comment:
Those two includes appear to be unused? Also, `#include <format>` causes the Linux build to fail as far as I can tell.

Reply:
oh shit, yeah I'm not even using format i think vscode added that while i wasn't paying attention